Homework4/Homework4/LibraryForMapping.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Homework4;
+
+public static class LibraryForMapping
+{
+    public static void MapTo<TSource, TDestination>(this TSource source, TDestination destination)
+    {
+        var sourceProperties = typeof(TSource).GetProperties();
+        var destinationProperties = typeof(TDestination).GetProperties();
+
+        foreach (var sourceProperty in sourceProperties)
+        {
+            var destinationProperty = destinationProperties.FirstOrDefault(x => x.Name == sourceProperty.Name);
+            if (destinationProperty != null)
+            {
+                destinationProperty.SetValue(destination, sourceProperty.GetValue(source));
+            }
+        }
+    }
```

Comment:
Can you also add addition methods to map only specific properties?

Reply:
Fixed